components/ActionSheet/src/MDCActionSheetController.h

```diff
@@ -18,6 +18,21 @@
 #import "MaterialElevation.h"
 
 @class MDCActionSheetAction;
+@class MDCActionSheetController;
+
+/**
+ MDCActionSheetControllerDelegate provides a method that allows a delegate of an
+ MDCActionSheetController to respond to its dismissals.
+ */
+@protocol MDCActionSheetControllerDelegate <NSObject>
+@optional
+
+/**
+ This method allows a delegate conforming to MDCActionSheetControllerDelegate to respond to
```

Comment:
Nit: `Tells the delegate the @c actionSheetController was dismissed.`

Reply:
Done